client-report/Dockerfile

```diff
@@ -23,6 +22,8 @@ COPY . .
 # GIT_HASH is optional and will be set properly when running `make` (see Makefile).
 # Or may be passed in at build time with --build-arg GIT_HASH=$(git rev-parse --short HEAD)
 ARG GIT_HASH
+ARG SERVICE_URL
 ENV GIT_HASH ${GIT_HASH:-placeholder}
+ENV SERVICE_URL ${SERVICE_URL}
 
-CMD npm run deploy:prod
+CMD npm run build:prod
```

Comment:
Need to circle back around and make sure that cache files are getting tagged for cache busting; This was one thing that happened in `deploy:prod` but not `build:prod` on some of the clients.